src/ci/citool/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1"
+askama = "0.13"
```

Comment:
Discussion: should this be gated behind a feature, or is the intention that the dashboard might be usable even in CI? Or rather, is it even worth the maintenance effort to gate it behind a feature?

Reply:
I think that eventually we might want to run it also on CI and then upload the generated website to S3 or something, to just provide a link in the post-merge report, so that users don't have to run it locally.

I wanted to feature gate advanced dependencies (https://github.com/rust-lang/rust/pull/139678), but we decided to use a cache on CI instead to make the build faster.

Reply:
Fair enough, that sounds like a good plan